app/services/albumService.js

```diff
@@ -35,3 +35,14 @@ exports.create = (userId, title, albumId) => {
     throw errors.databaseError;
   });
 };
+
+exports.getAlbumsMe = userId => {
```

Comment:
As the function is receiving a userId, the name could be something like `getAlbumsForUserId`